assembl/static2/js/app/pages/survey.jsx

```diff
@@ -17,16 +18,58 @@ import { getIfPhaseCompletedByIdentifier } from '../utils/timeline';
 import ThematicQuery from '../graphql/ThematicQuery.graphql';
 import { displayAlert } from '../utils/utilityManager';
 
-class Survey extends React.Component {
+type PostNode = {
+  node: {
+    id: string,
+    originalLocale: string
+  }
+};
+
+type QuestionType = {
+  id: string,
+  posts: {
+    edges: Array<PostNode>
+  },
+  title: string
+};
+
+type SurveyProps = {
+  debate: {
+    debateData: {
+      timeline: string
+    }
+  },
```

Comment:
We should really look at integrating graphql object types with flow instead of writing manually the types here.

Reply:
yes ! https://bluenove.atlassian.net/browse/DEVAS-949